.github/workflows/simple.yml

```
@@ -17,6 +17,6 @@ jobs:
     - name: Configure
       run: cmake -S . -B build -DGOOFIT_DEVICE=OMP -DGOOFIT_CXX_STANDARD=14
     - name: Build
-      run: cmake --build build -j10
+      run: cmake --build build -j"$(getconf _NPROCESSORS_ONLN)"
```

I'm sorry for the mistake in my first suggestion. As a far as I know a space between `-j` and `"$(getconf _NPROCESSORS_ONLN)"` is mandatory. So, correct command should be as follows:
```suggestion
      run: cmake --build build -j "$(getconf _NPROCESSORS_ONLN)"
```